apis/infrastructure/v1beta1/byohost_types.go

```diff
@@ -32,6 +32,26 @@ type ByoHostSpec struct {
 	BootstrapSecret *corev1.ObjectReference `json:"bootstrapSecret,omitempty"`
 }
 
+// HostStatus is a set of details about the host platform.
+type HostStatus struct {
```

Comment:
Oh! Just realized that `HostInfo` is also a struct used during registration ([host_register.go](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/blob/main/agent/registration/host_registrar.go#L26)). But this will be removed by #477.

Reply:
Oh ok, ^^ type will be introduced in [byohost_types](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/blob/main/apis/infrastructure/v1beta1/byohost_types.go) so it should be okay.

Reply:
Thanks for highlighting this. I guess it should be okay.